psf_proposal-SCiCookie2.md

```diff
@@ -0,0 +1,144 @@
+# Proposal for Python Software Foundation
+
+## Form
+
+- Contact
+
+  - Legal First Name*: Ivan
```

Comment:
I think that for now it is ok to keep my name here, but as you are maintainers as well, it would be ok to have Anavelyz's or Yurely's name (not both) here.